internal/cmd/keys.go

```diff
@@ -174,6 +175,16 @@ func newKeysListCmd() *cobra.Command {
 				}
 			}
 
+			machines, err := client.ListIdentities(api.ListIdentitiesRequest{})
+			if err != nil {
+				return err
+			}
```

Comment:
Querying in batches can help, but it seems like it's shifting the problem around.

Instead of more API requests, could we augment the response of `ListAccessKeys` to include the name we need? We can query the extra info from the DB before returning the request, which I think is a nicer API, and addresses the performance concerns of returning too much data.